tests/function_corpus.py

```diff
@@ -267,7 +267,19 @@ def make_shape(dims, length):
 
     return target_shape.flatmap(generate_source)
 
-def broadcasted_elementwise_strategy(k: int):
+def non_zero_float_strategy(min_value=-10.0, max_value=10.0, epsilon=1e-3):
+    @st.composite
+    def strategy(draw):
+        x = draw(st.floats(
+            allow_infinity=False,
+            allow_nan=False,
+            min_value=min_value,
+            max_value=max_value
+        ))
+        return x + (epsilon if x >= 0 else -epsilon)
+    return strategy()
+
+def broadcasted_elementwise_strategy(k: int, values=None):
```

Comment:
using elements=None isn't ideal. I want -10 to 10 to be the default. When None is used we might get huge massive values which can lead to some numerical issues that I don't want to deal with right now.